### ISSUE TITLE
[Enhancement] Reuse of access token before the token expiration time

### ISSUE THREAD

**Issue by twentyTwo** [docusign/sfdc-recipe-rest-create-envelope#3]
While creating an envelope, each time a new access token is generated in the current version.

```
public static void createEnvelope() {
        //Get the access token
        String accessToken = getAccessToken();
        .......
}
```
What can be done is, save the **access token** and the **expiration time** in a custom object so that we can reuse the access token within the expiration time.

**Comment by LarryKlugerDS** [comment]
Yes, that is a good idea. If you implement this idea, please submit a pull request to update the example.

Thank you.

**Comment by twentyTwo** [comment]
@LarryKlugerDS : I have implemented the idea and tested it in my org. I have submitted a pull request. Please have a look, review and merge it if all is okay.

**Comment by LarryKlugerDS** [comment]
Thank you! I have submitted internal request for a review of your pull request.
